.github/stale.yml

```diff
@@ -0,0 +1,34 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 5
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+# type/discussion - our discussion pages. They can be inactive for a long time.
+# WIP - pull requests that have "work in progress" status
+exemptLabels:
+  - type/discussion
+  - WIP
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```

Comment:
```suggestion
issues:
  closeComment: >
     This issue is closing due to inactivity.
     In the case of bugs or questions, feel free to create new GitHub [issues](https://github.com/DevExpress/DevExtreme/issues) or tickets in our [Support Center](https://www.devexpress.com/Support/Center/).
```

Reply:
LGTM

@RomanTsukanov  could you check it?

Reply:
```suggestion
issues:
  closeComment: >
     This issue was closed due to inactivity.
     If you found a bug or have a question about our product, [create a new issue](https://github.com/DevExpress/DevExtreme/issues) on GitHub or submit a ticket in our [Support Center](https://www.devexpress.com/Support/Center/).
```